### PR TITLE
Phase4.2

### PR DESCRIPTION
**Pull request by mccurdy42**

### CODE REVIEW

File: second_cplex/second_cplex.mod [hunk 1]
```diff
@@ -34,14 +37,30 @@ range day3 = 13..18;
...
+}
 // Problem parameters are defined below
-int prep[r2]= [240,240,120]; //prep time array
 int lengtht[r4]= [40,60,50,50,60,40,
 				 40,60,50,50,60,40,
 				 40,60,50,50,60,40,
 				 40,60,50,50,60,40,
 				 40,60,50,50,60,40];   
+
 int teachMin[r2]=[1260,1260,630];
```
Comment (mestrona3) [inline second_cplex/second_cplex.mod]: dont see this array being used anymore because there is more teachers, this can be deleted?

Comment (mestrona3) [reply second_cplex/second_cplex.mod]: int teachMin[r2]=[1260,1260,630];
float FTE[r2]= [1.0,1.0,0.5];
float totalTeacherMin[r2];
int totalTime = 1500;

All need to be deleted as they are previously defined

**Comment by mccurdy42** [comment]
Hello! the items were removed . I added them inadvertently when resolving the merge conflicts. Thank you for catching them.

**Review by mestrona3 — APPROVED** [review]
WOWOWOW LOVE IT

**Pull request merged**